core/domain/email_manager_test.py

```diff
@@ -2054,6 +2058,1419 @@ def test_that_correct_completion_email_is_sent(self):
                 feconf.EMAIL_INTENT_REVIEW_SUGGESTIONS)
 
 
+class NotifyContributionDashboardReviewersEmailTests(test_utils.EmailTestBase):
+    """Test that correct email is sent while notifying Contributor Dashboard
+    reviewers.
+    """
```

Comment:
```suggestion
    """Tests that the correct email is sent while notifying Contributor Dashboard
    reviewers in different situations.
    """
```

Not sure I like my suggested wording, but the documentation just seemed to need _some_ changes.

Reply:
haha fair..I changed it